build/write_bandreps_from_bandrepset.jl

```diff
@@ -50,11 +50,11 @@ function bandrep2csv(io::IO, BRS::BandRepSet)
 end
 
 # Load ::BandRepSet for all 2D plane groups
-include("tbd-filename.jl") # should define BRS_vec
+include("Band_Representations.jl") # should define BRS_vec
```

Comment:
I try to stick to a lower-case convention for file names (except for the main package file, which has to be the same as the package name), so maybe something like `band-representations.jl` here instead?